assets/src/edit-story/components/canvas/useInsertElement.js

```diff
@@ -160,6 +160,10 @@ function getElementProperties(
   if (!isNum(y)) {
     y = 0;
   }
+
+  // When dragging an element to canvas, this means that the element will always be
+  // positioned inside the design space, even if the user drags it partially outside.
+  // @todo Is this intended behavior?
```

Comment:
I don't think so, no (that this is intended behavior). I think it's fair to say that it should at least be partially on canvas, but not fully. Removing `- width` and `- height` respectively would fix that.